docs/src/components/RightSidebar/RightSidebar.astro

```diff
@@ -0,0 +1,25 @@
+---
+import TableOfContents from './TableOfContents.jsx';
+import MoreMenu from './MoreMenu.astro';
+const {content, githubEditUrl} = Astro.props;
+const headers = content.astro.headers;
+---
+<style>
+  .sidebar-nav {
+    width: 100%;
+    position: sticky;
+    top: 0;
+  }
+  .sidebar-nav-inner {
+    height: 100%;
+    padding: 0;
+    padding-top: var(--doc-padding);
+    overflow: auto;
+  }
+</style>
+<nav class="sidebar-nav" aria-labelledby="sidebar-content">
```

Comment:
this aria-labelledby needs to be updated to the id of the element that's wrapping it in MainLayout.astro

Reply:
great catch